hack/go-linter.sh

```diff
@@ -4,20 +4,23 @@ set -e
 source ./hack/common.sh
 
 fetch_go_linter
-
+    # The following arg ` -e "internal value || yum "` was added to ignore any line
+    # which has the text `internal value`  or `yum`. More info: https://github.com/golangci/golangci-lint#command-line-options
 header_text "Running golangci-lint"
 golangci-lint run --disable-all \
     --deadline 5m \
     --enable=nakedret \
     --enable=maligned \
     --enable=ineffassign \
     --enable=goconst \
+    --enable=lll  \
+    --exclude [a-z] \
     --enable=goimports \
 
+
 ##todo(camilamacedo86): The following checks requires fixes in the code
 # --enable=golint
```

Comment:
See that it was refac and now the impl is in `./hack/tests/check-lint.sh`
Where is required remove from the dev linters and add in the genaral/common execution [here](https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/check-lint.sh#L47) to enable this check for the ci.

Reply:
ok